constants/mappers.ts

```diff
@@ -0,0 +1,17 @@
+export const API_MAPPERS = {
+  mapDaoDTOtoDao: 'mapDaoDTOtoDao',
```

Comment:
I consider we should write inner fields of constants in SCREAMING_SNAKE_CASE. @VERZUOL1 @karpovdmitry WDYT?

Reply:
Agree. This is the format we use across the app.

Reply:
@VERZUOL1 @AlexeyPonomarenko fixed